src/index.js

```diff
@@ -1,17 +1,61 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import Map from './components/Map'
+
+class Home extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            currentLocation: {
+                latitude: null,
+                longitude: null
+            }
+        };
+
+    }
+
+    componentDidMount() {
```

Comment:
In a function-based component, we'd do a `useEffect()` here; if you haven't yet read through [the article on useEffect](https://overreacted.io/a-complete-guide-to-useeffect/) that's been going around, I definitely recommend it.

In a client project, we'd probably set this up to push the location information into the Redux state rather than hold it in component state because it's kind of a global thing, but that's not really a big deal here.